Source/WebCore/platform/graphics/gstreamer/GStreamerVideoSinkCommon.cpp

```diff
@@ -55,6 +55,10 @@ class WebKitVideoSinkProbe {
                     GST_DEBUG_OBJECT(pad, "FLUSH_START received, aborting all pending tasks in the player sinkTaskQueue.");
                     self->m_isFlushing = true;
                     player->sinkTaskQueue().startAborting();
+#if USE(GSTREAMER_GL)
```

Comment:
USE(GSTREAMER_GL) was removed in [282892@main](https://commits.webkit.org/282892@main) before this patch landed, so this code is effectively disabled at build time.

Reply:
Addressed in https://github.com/WebKit/WebKit/pull/40629 ...